<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsIOScreen</class>
 <widget class="QWidget" name="SettingsIOScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sensorButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
	color: #000000;
}

QPushButton:pressed{

	background-color: #006064;
}

QPushButton:checked{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>Sensor</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="verticalLine1">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>0</y>
     <width>1</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="heatingButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
	color: #000000;
}

QPushButton:pressed{

	background-color: #006064;
}

QPushButton:checked{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>Heating</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="verticalLine2">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>0</y>
     <width>1</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="lightingButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
	color: #000000;
}

QPushButton:pressed{

	background-color: #006064;
}

QPushButton:checked{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>Lighting</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="verticalLine3">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>0</y>
     <width>1</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="miscButton">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
	color: #000000;
}

QPushButton:pressed{

	background-color: #006064;
}

QPushButton:checked{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>Misc.</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="hotizontalLine1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>739</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
